content/search_api_3/latest/searching/offers/status.md

```diff
@@ -0,0 +1,60 @@
+---
+---
+
+# Status
+
+The status indicates whether an offer is still available for visits or not. The following values for ‘status’ are applicable:
+
+| Value | Meaning (events) | Meaning (places) |
+|--|--|--|
+| `Available` | the event takes place as planned | the place is ‘open’ and can be visited during opening hours | 
+| `TemporarilyUnavailable` | the event has been postponed to a later date yet to be determined | the place is temporarily closed (due to renovations for example) | 
+| `Unavailable` | the event has been cancelled | the place still exists (physically), but is permanently close | 
+
+You can filter by address-related fields using two methods:
+* URL parameter
+* Advanced queries
+
+## URL parameter
+
+Using the `status` URL parameter, you can limit the results you get back to events with a certain status.
+Example to get results that have the status `Available`:
+
+```
+GET https://search.uitdatabank.be/offers/?status=Available
+```
+
+Additionally, you can limit the results to documents that have either one of multiple status values.
+Example to get results that have the status `Unavailable` or `TemporarilyUnavailable`:
+
+```
+GET https://search.uitdatabank.be/offers/?status=Unavailable,TemporarilyUnavailable
+```
+
+If used on its own, it will filter on the top-level status of the events/places.
+
+If used in combination with `dateFrom` and/or `dateTo` (see [date & time](https://documentatie.uitdatabank.be/content/search_api_3/latest/searching/offers/date.html)) it will combine the given status(es) with the given dateFrom and/or dateTo to query on subEvents, so you'll only get results that actually have the given status(es) during the given date range.
```

Comment:
Should we also document that it also does this for `localTimeFrom` and `localTimeTo`?

Reply:
Yes, will add it after the localTime docs are merged.

Reply:
Perfect 👍